addons/stock/wizard/stock_backorder_confirmation.py

```diff
@@ -46,7 +46,7 @@ def _check_less_quantities_than_expected(self, pickings):
                                  precision_rounding=move.product_uom.rounding) > 0:
                     moves_to_log[move] = (move.quantity_done, move.product_uom_qty)
             if moves_to_log:
-                pick_id._log_less_quantities_than_expected(moves_to_log)
+                pick_id.sudo()._log_less_quantities_than_expected(moves_to_log)
```

Comment:
It's better to add 'sudo' only when necessary, so I suggest changing it here: https://github.com/odoo/odoo/blob/16.0/addons/sale_stock/models/stock.py#L152"

Reply:
@DjamelTouati Fixed it, thanks